docs/pricing/index.mdx

```diff
@@ -76,7 +76,35 @@ Tracing is enabled by and will be billed for in spans.
 
 #### Cron Monitors Pricing
 
-Monitors for Crons can't be purchased in advance. Crons are available to use and purchase through a pay-as-you-go budget that can be shared across all event types.
+All Sentry plans include **one cron monitor**. To activate additional monitors, set up a **Pay-As-You-Go (PAYG) budget**. Additional monitors can't be purchased in advance; they are only available through your PAYG budget, which can be shared across all event types.
+
+**Key Points:**
+
+- **Deactivating or Deleting Monitors**:
+  - Deactivated or deleted monitors will **count** towards your billing quota if they were previously active in the current billing period. Otherwise, they don't count towards your billing quota.
```

Comment:
```suggestion
  - Deactivated or deleted monitors will **count** towards your billing quota if they were previously active in the current billing period. Otherwise, they won't count towards your billing quota.
```